src/driver/iour/mod.rs

```diff
@@ -29,6 +34,9 @@ pub struct Driver {
 }
 
 impl Driver {
+    /// Reserved code for async cancel operation
+    pub const CANCEL: u64 = u64::MAX;
```

Comment:
Nice catch!

Reply:
Maybe it makes sense to define cross-platform reserved opcode range near u64::MAX if there will be more reserved codes.